src/pages/edge/edge-network/tutorials/datastream-config-override-rules.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Datastream config override for Edge events using Rules.
+description: This guide provides instructions on overriding datastream ID and datastream configuration for Edge Network Requests that are not initiated by the sendEvent API, all accomplished through Tags Rules.
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Datastream config override for Edge events using Rules
+
+By default, all **Edge Experience events** are sent to the datastream configuration specified in the **Adobe Experience Platform Edge Network** extension configuration settings. This tutorial will offer examples demonstrating how to customize this configuration on a per-event basis
+
+<InlineAlert variant="info" slots="text"/>
+
+Starting with `Edge` extension version **2.4.0** (Android) and **4.3.0** (iOS), the `sendEvent` API supports optional datastream overrides. For detailed information, please refer to the [sendEvent API reference](./api-reference.md#sendEvent).
+
+## Steps to add datastream config overrides for Lifecycle Edge events using rules
+
+Let's add **datastream ID override** and **datastream config overrides** for `Lifecycle Edge` events of type **application.launch** and **application.close**:
+
+### 1. Add a new rule for your mobile property
+
+Give your rule an easily recognizable name in your list of rules.
+In this example, the rule is named "Attach datastream config overrides Data to Lifecycle Edge events".
+
+---
```

Comment:
Nice split on the sections, the content flows nicely! With this new structure I think you can also skip these pause breaks, so they only show for the top headers